springwolf-add-ons/springwolf-generic-binding/src/test/java/io/github/springwolf/addons/generic_binding/annotation/processor/AsyncGenericOperationBindingProcessorTest.java

```diff
@@ -136,5 +159,17 @@ void yamlSyntaxDoesWorkAsWell() {
             // then
             assertThat(result).isEqualTo(Map.of("key", "value"));
         }
+
+        @Test
+        void yamlSyntaxArrayPropertyTest() {
+            // given
+            String[] strings = {"key: [value1, value2, value3]"};
```

Comment:
Comment: looks fine based on the yaml standard

I wonder whether the values should be escaped ['value1'], but that can be a future task/feature request

Reply:
A quick test shows that we can do `"key: [value1, value2, value3 is long]"` and it works fine. The moment we add `'` we need to provide extra escaping support.

We can do it, but I wonder if would make more sense to modify the annotation and provide something like
```
@AsyncGenericOperationBinding(
      type = "test-binding",
      yaml = """
                  something:
                     - 'long description'
                     - 'something else'
                  """
```
and use a YAML Parser for the field. That would provide complete support for any combination.

Reply:
Totally fine. Just something I thought about. In case it becomes an issue, we can address it then